requirements-dev.txt

```diff
@@ -177,10 +179,6 @@ funding-service-design-utils==2.0.32
     # via -r requirements.txt
 govuk-frontend-jinja==2.8.0
     # via -r requirements.txt
```

Comment:
It looks like it's already `2.8.0` - or is this a problem with the diff?

Reply:
Thanks, reverted the unnecessary changes to requirements files.